src/Project/JobRunner.ts

```diff
@@ -71,8 +104,11 @@ export class JobRunner extends EventEmitter {
       this.logger.outputWithTime('Finish Running ONE compilers.');
       this.emit(K_CLEANUP);
       return;
+    } else if (job.jobType >= Job.Type.tInstall && job.jobType >= Job.Type.tInstalled) {
```

Comment:
```suggestion
    } else if (job.jobType >= Job.Type.tInstall && job.jobType <= Job.Type.tInstalled) {
```

Reply:
You don't have to care about this. This will be handled more clearly soon.

Reply:
That's why Job has callbacks.